scripts/tasks/sandbox-parts.ts

```diff
@@ -61,6 +63,16 @@ export const create: Task['run'] = async ({ key, template, sandboxDir }, { dryRu
       throw new Error(`Missing repro directory '${srcDir}', did the generate task run?`);
     }
     await copy(srcDir, sandboxDir);
+
+    // If the template is pnpm-based, we need to add the `packageManager` entry to the package.json
+    // because otherwise the pnpm CLI rejects its usage, since it finds yarn configured higher up the tree.
+    if (isPnpmTemplate(template.name)) {
+      const packageJsonPath = join(sandboxDir, 'package.json');
+      const packageJson = await readJson(packageJsonPath);
+      logger.info('📝 Adding packageManager entry to package.json');
+      packageJson.packageManager = 'pnpm@8.6.11';
```

Comment:
add a comment explaining that this version is not special, it can be changed to the latest whenever we want